openmp/runtime/src/kmp_dispatch.cpp

```diff
@@ -90,6 +90,69 @@ static inline int __kmp_get_monotonicity(ident_t *loc, enum sched_type schedule,
   return monotonicity;
 }
 
+#if KMP_WEIGHTED_ITERATIONS_SUPPORTED
+static inline float __kmp_get_float_val(float num) {
```

Comment:
Add a comment, get_float_val float->float is really not explaining much

Reply:
Added comment, and changed function name for clarity.